pages/vi/vi-bellapps.md

```diff
@@ -65,9 +65,9 @@ We created a course for aspiring virtual interns so that we could test the softw
 Take the course, and the last question will ask you to specify any problems or improvements/suggestions that you have. Remember, people taking these courses in remote areas of the world will probably run into the same problems if you do not raise them now and let the team know. This is a very important task, and your help is much appreciated.
 
 **STEPS:**
-1. Click on Courses on the top navigation
-2. Find the course "Virtual Interns," click on Admission
-3. Click on Open
+1. Click on `Courses` on the top navigation
+2. Find the course "Virtual Interns," click on `Admission`
+3. Click on `Open`
 4. Carefully finish 12 steps of the course
```

Comment:
I feel like *12 steps* should also be `12 steps`

Reply:
@ Sriharsha-Singam I felt to highlight the clickable or changeable elements as it was done below of that description.

![image](https://user-images.githubusercontent.com/25775298/42791677-70db45fa-8937-11e8-8eec-0dc8a67dc52b.png)

Reply:
@BhargaviNadendla Seems fair.

Reply:
ok